padrino-helpers/lib/padrino-helpers/render_helpers.rb

```diff
@@ -34,7 +34,7 @@ def partial(template, options={}, &block)
         options = { :locals => {}, :layout => false }.update(options)
         explicit_engine = options.delete(:engine)
 
-        path,_,name = template.to_s.rpartition(File::SEPARATOR)
+        path, name = File.split(template.to_s)
         template_path = File.join(path,"_#{name}").to_sym
```

Comment:
ref: https://github.com/middleman/middleman/pull/1975
ref: https://github.com/middleman/middleman/blob/07651c63a693bff183af907d274641d08f598565/middleman-core/lib/middleman-core/template_context.rb#L132

Please note that middleman usage of `#partial` was unexpected and template_path was patched by tinkering with it's beginning '/'.

Fixed both issues in 2aa518dc2fcc6df1c28e03b8f428a1d6af362d9d

Please respond if this fix is acceptable and I will push out a Padrino bugfix release '0.13.3.1'.